src/index.js

```diff
@@ -180,10 +180,10 @@ export default {
             
             if (deviceId.length === 0) continue; // not a device
             debug.log('[android] found device: ' + deviceId);
-            const deviceModel = (await this._shellExec(`adb -s ${deviceId} shell getprop ro.vendor.product.model`))[0];

```

Comment:
Is the change between vendor.product and product.vendor documented anywhere? Or is it just a plain bug

Reply:
Ah, I just googled it and find 
`BUILD_LEGACY_PROPERTIES.put("ro.product.vendor.model", "ro.vendor.product.model");` from

https://cs.android.com/android/platform/superproject/+/master:test/suite_harness/common/host-side/tradefed/src/com/android/compatibility/common/tradefed/targetprep/VtsDeviceInfoCollector.java;l=86

Also when I run `adb shell getprop` to print all properties of the emulator on my local, I can only find `ro.product.vendor.model`. I believe this issue exists 2 years ago when the package is first time used in our project and at that time we use `testcafe 'android::$serial_number' tests` to start the test. As the deviceModel returns empty value due to not found, double `colon` is used to make it work.

Interesting finding in Android mock files that `ro.product.vendor.model` is not always defined, but `ro.product.model` is. https://cs.android.com/android/platform/superproject/+/master:external/cpuinfo/test/mock/galaxy-s5-global.h